src/lib/optimizer/optimizer.cpp

```diff
@@ -37,6 +37,8 @@ namespace opossum {
 std::shared_ptr<Optimizer> Optimizer::create_default_optimizer() {
   auto optimizer = std::make_shared<Optimizer>();
 
+  // Run ExpressionReductionRule, but do not yet add new between nodes (only replace existing like predicates with
+  // between if applicable).
```

Comment:
> but do not yet add new between nodes (only replace existing like predicates with between if applicable)

I think this description is a bit unclear to those unfamiliar with the various LIKE transformations in the ExpressionReductionRule. Maybe write that one specific LIKE transformation is disabled, which may add PredicateNodes with BetweenExpressions.